sdk/python/kfp/v2/google/aiplatform.py

```diff
@@ -237,15 +235,14 @@ def _get_custom_job_op(
               path=path,
           ) for input_name, path in dummy_inputs.items()
       ],
-      task_outputs=dummy_outputs
-  )
+      task_outputs=dummy_outputs)
 
 
 def custom_job(
     name: str,
     input_artifacts: Optional[Dict[str, dsl.PipelineParam]] = None,
     input_parameters: Optional[Dict[str, _ValueOrPipelineParam]] = None,
-    output_artifacts: Optional[Dict[str, Type[artifact.Artifact]]] = None,
+    output_artifacts: Optional[Dict[str, Type[io_types.Artifact]]] = None,
```

Comment:
I think that "artifact" or "artifact_types" module name is better than "io_types". When I see "io_types" I think "File-based I/O", "URI-based I/O", "Network I/O", etc.

Reply:
@neuromage what do you think about the naming?

As we're deleting the current `dsl/artifact.py`, do we want to rename `io_types.py` to `artifact.py`?
Alternatively, we can promote `Artifacts`, `Model`, etc. to the dsl level (via `__init__.py`), so user would use `dsl.Artifact`, `dsl.Model`,  etc., and the file name becomes an implementation detail.

Reply:
Let's keep this as is for now and revisit post-cleanup.